CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
 - Modify Perseo's executable to add environment variable-based configuration.
+- Add string subsitution for more action parameters
```

Comment:
Given that in this case we have an issue for that feature (#114), let's incluce it in the CNR entry.